app/routes/login.tsx

```diff
@@ -15,10 +16,31 @@ export const action = async ({ request }: ActionArgs) => {
 
 export default function Login() {
   return (
-    <Form method="post">
-      <Heading>Login</Heading>
-      <Text>You will be logged in (a session will be created for you)</Text>
-      <Button type="submit">Login</Button>
-    </Form>
+    <Center height="container.md" marginTop="16">
+      <Grid gap="12">
+        <GridItem width="lg">
+          <VStack marginTop="20" spacing="10">
```

Comment:
better use theme spacings, instead of putting exact values in pixels:
https://chakra-ui.com/docs/styled-system/style-props#margin-and-padding

So instead of `marginTop="20"` it would be something like `mt="4"`

Reply:
Are you perhaps referring to adding onto the theme folder with predetermined values as to the default provided by chakra? I never specified the exact pixel, I only used the values that is set on default. 

![image](https://user-images.githubusercontent.com/32577022/218175483-b2b8dc88-382e-47db-b874-931accaad2b9.png)

Reply:
Chakra already has a pretty decent list of predefined values, but of course, we could extend it if we need to

Reply:
I am using their predefined values, I'm a bit confused as to what I did wrong? `marginTop='20'` is similar to `marginTop='5rem'. It's a default value set out by chakra

Reply:
Oh, right, I just got confused, because supposed that a fully-written attributes, such as `marginTop` get pixels, while if you need to use theme value you put shortcut like `mt`. My bad, than all good, I would just recommend to use shortcuts to avoid making huge vertical attributes list

Reply:
I only use the full word for readability...